lstchain/datachecks/containers.py

```diff
@@ -155,7 +155,7 @@ def fill_event_wise_info(self, subrun_index, table, mask, geom,
                                       frame=AltAz(obstime=time_utc,
                                                   location=location))
         self.tel_ra = telescope_pointing.icrs.ra.value
```

Comment:
I think this does the `altaz` → `icrs` transformation twice unnecessarily. Transform once, then use the result.

Reply:
It is just 2 calls per subrun, and I think it is more compact like this...